src/scripts/num.js

```diff
@@ -139,6 +139,12 @@ export function uatoms(number = 0) {
     .toString()
 }
 
+export function gasMilliUnits(number = 0) {
+  return BigNumber(number)

```

Comment:
why is this working? this is just removing the fixed part. but milliunits should never have decimals

Reply:
because we are sending them as decimals. Should I change the name or should we change the format and use proper milliunits for gas fees? 

As an user I think it is easier to understand decimals, since it is the same unit they are using for their transaction and therefore doesn't lead to confusion.

I need to change the naming then

Reply:
this is used to convert view value to milli unit values, right?

Reply:
No. That is what is confusing. It is simply doing `times(1e6)`. That's is all it has to do.

I'm changing the name